packages/aws-cdk-lib/aws-ec2/test/example.images.lit.ts

```diff
@@ -3,16 +3,31 @@ import * as ec2 from '../lib';
 /// !show
 // Pick the right Amazon Linux edition. All arguments shown are optional
 // and will default to these values when omitted.
-const amznLinux = ec2.MachineImage.latestAmazonLinux({
-  generation: ec2.AmazonLinuxGeneration.AMAZON_LINUX,
+const amznLinux2 = ec2.MachineImage.latestAmazonLinux2({
   edition: ec2.AmazonLinuxEdition.STANDARD,
   virtualization: ec2.AmazonLinuxVirt.HVM,
   storage: ec2.AmazonLinuxStorage.GENERAL_PURPOSE,
   cpuType: ec2.AmazonLinuxCpuType.X86_64,
+  kernel: ec2.AmazonLinux2Kernel.KERNEL_5_10,
+});
+
+const amznLinux2023 = ec2.MachineImage.latestAmazonLinux2023({
+  edition: ec2.AmazonLinuxEdition.STANDARD,
+  cpuType: ec2.AmazonLinuxCpuType.X86_64,
+  kernel: ec2.AmazonLinux2023Kernel.KERNEL_6_1,
 });
```

Comment:
This can be moved to another PR, just noticed the deprecated method usage when updating the Windows example below